src/app/subscriptions.elm

```diff
@@ -11,5 +10,12 @@ pollServer =
 
 
 subscriptions : Model -> Sub Msg
+
+
+
+-- TODO: Replace Sub.none with a WebSocket subscription to "ws://" ++ pollServer ++ "/socket"
```

Comment:
Same I think we shouldn't specify `"ws://" ++ pollServer ++ "/socket"` they can find out by themselves how to concatenate string in ELM